Src/DependencyCollector/NuGet/Package.nuspec

```diff
@@ -49,9 +49,9 @@
     <file src="$configuration$\Src\DependencyCollector\Net45\Microsoft.AI.DependencyCollector.pdb" target="lib\net45" />
     <file src="$configuration$\Src\DependencyCollector\Net45\Microsoft.AI.DependencyCollector.xml" target="lib\net45" />
 
-    <!-- For .NET Standard 1.6 projects -->
-    <file src="$configuration$\Src\DependencyCollector\NetCore\netstandard1.6\Microsoft.AI.DependencyCollector.dll" target="lib\netstandard1.6" />
-    <file src="$configuration$\Src\DependencyCollector\NetCore\netstandard1.6\Microsoft.AI.DependencyCollector.pdb" target="lib\netstandard1.6" />
-    <file src="$configuration$\Src\DependencyCollector\NetCore\netstandard1.6\Microsoft.AI.DependencyCollector.xml" target="lib\netstandard1.6" />
+    <!-- For .NET Standard 1.3 projects -->
+    <file src="$configuration$\Src\DependencyCollector\NetCore\netstandard1.3\Microsoft.AI.DependencyCollector.dll" target="lib\netstandard1.3" />
+    <file src="$configuration$\Src\DependencyCollector\NetCore\netstandard1.3\Microsoft.AI.DependencyCollector.pdb" target="lib\netstandard1.3" />
+    <file src="$configuration$\Src\DependencyCollector\NetCore\netstandard1.3\Microsoft.AI.DependencyCollector.xml" target="lib\netstandard1.3" />
```

Comment:
So, we're building netstandard 1.5, but not actually including it in the NuGet? Why not?